src/nbl/asset/utils/CHLSLCompiler.cpp

```diff
@@ -123,7 +140,7 @@ CHLSLCompiler::DxcCompilationResult CHLSLCompiler::dxcCompile(std::string& sourc
 
     result.objectBlob = resultingBlob;
 
-    return result;
+    return { (uint8_t*)result.objectBlob->GetBufferPointer(), result.objectBlob->GetBufferSize() };
```

Comment:
you do realize a com-pointer is a smart pointer and it will go out of scope here?

go back to how the functions were please (this wasn't a method of CHLSLCompiler)